src/main/java/com/hillel/items_exchange/service/AdvertisementService.java

```diff
@@ -116,6 +119,26 @@ public void remove(long id) {
         advertisementRepository.flush();
     }
 
+    public void setAdvertisementDefaultImage(Long advertisementId, Long imageId, User owner) {
+        final Advertisement advertisement = owner.getAdvertisements().stream()
+                .filter(adv -> adv.getId() == advertisementId)
+                .findFirst().orElseThrow(() -> new IllegalIdentifierException(
+                        getExceptionMessageSourceWithAdditionalInfo(
+                                "exception.illegal.id",
+                                "User does not has advertisement with such id"
```

Comment:
Unfortunately, additional info isn't translated in case russian localization. I think you shouldn't use `getExceptionMessageSourceWithAdditionalInfo` in such way.